### ISSUE TITLE
[Request] Add Return Value for openFileSelector

### ISSUE THREAD

**Issue by ZhUyU1997** [Jaaneek/useFilePicker#56]
Why not return a promise from openFileSelector?
We can direct get results from it.

```jsx
<button onClick={async () => {
  const result = await openFileSelector()
  console.log(result)
}}>Select files </button>
```

**Comment by MrKampla** [comment]
Great suggestion! I'll prepare a PR with proposed change soon.

**Comment by MrKampla** [comment]
Unfortunately, it is not possible to implement that feature reliably with the current browser APIs. We tried several solutions, but we weren't able to find one that was working across all platforms. There's more explanation here: https://github.com/Jaaneek/useFilePicker/pull/58#issuecomment-1496680066
Instead of promise, we decided to add onChange handlers for successful, rejected and any file selections (both correct and incorrect). This feature is being implemented in this PR: #60 and it's going to be included in the next release. 
Thanks for your suggestion @ZhUyU1997
